_posts/2016-12-06-crystal-0.20.1-released.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Crystal 0.20.1 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.20.1](https://github.com/crystal-lang/crystal/releases/tag/0.20.1) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements - in [75 commits since 0.20.0](https://github.com/crystal-lang/crystal/compare/0.20.0...0.20.1).
+
+# Exciting Changes
+
+* `p` and `pp` use pretty printing.
+* `finished` macro hook was added. It places it's contents at the end of the program when all other types and methods are known.
+
+<div class="code_section">{% highlight ruby %}
+class MyModel
+  MAPPING = {
+    id: Int32
+  }
+
+  macro attributes(**values)
```

Comment:
`extra_attributes`

also this example seems a tad convoluted, not sure an example works well for this entry.

Reply:
Well it certainly helps to be illustrated but I'm not so confident with that example myself.